frontend/src/components/Diff/Diff.tsx

```diff
@@ -24,19 +24,20 @@ function FormatDiffText({ texts }: { texts: api.DiffText[] }) {
     } </>
 }
 
-function DiffColumn({ diff, prop, header, className }: {
+function DiffColumn({ diff, prop, header, className, hoveredSourceLine }: {
     diff: api.DiffOutput | null
     prop: keyof api.DiffRow & keyof api.DiffHeader
     header: ReactNode
     className?: string
+    hoveredSourceLine?: number | null
 }) {
     return <resizer.Section className={classNames(styles.column, className)} minSize={100}>
         <div className={classNames(styles.row, styles.header)}>
             {header}
         </div>
         <div className={styles.body}>
             {diff?.rows?.map?.((row, i) => (
-                <div key={i} className={styles.row}>
+                <div key={i} className={(typeof row[prop]?.src_line != "undefined" && row[prop]?.src_line == hoveredSourceLine) ? styles.hoveredSourceRow : styles.row}>
```

Comment:
```suggestion
                <div
                    key={i}
                    className={classNames({
                        [styles.row]: true,
                        [styles.highlight]: typeof hoveredSourceLine == "number" && row[prop]?.src_line == hoveredSourceLine}
                    })}
                >
```

Reply:
You may need to `import classNames from "classnames"`

Reply:
I haven't really changed anything here regarding classNames though...